all.json

```
@@ -887,6 +887,7 @@
     "wallenmexico.com",
     "wallerhof.com",
     "wallet-api.link",
+    "walletauthenticatorapp.com"
```

```suggestion
    "walletauthenticatorapp.com",
```

Linting complains here.